new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -83,6 +84,8 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Test to create model in image domain and start the domain")
 @ExtendWith(Timing.class)
+@ExtendWith(IntegrationTestWatcher.class)
+@ITNamespaces(numofns = 2)
```

Comment:
numofns is kind of cryptic.  Is there a reason not to make this more verbose?

Reply:
Just wanted to keep the name shorter. Is "namespacesRequired" is good enough ?

Reply:
can't it just be `Namespaces(2)` ?

Reply:
Removed the annotation at class level and moved it to method parameter per Derek's comments. 
Also renamed it as Namespaces with default field. So method signature can use Namespaces(2)